samples/bluetooth/st_ble_sensor/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _bluetooth-st_ble_sensor:
+
+Bluetooth: ST BLE Sensor Demo
+#############################
+
+Overview
+********
+
+This application demonstrates BLE peripheral by exposing vendor-specific
+GATT services. Currently only button notification and LED service are
+implemented. Other BLE sensor services can easily be added.
+See `BlueST protocol`_ document for details of how to add a new service.
+
+Requirements
```

Comment:
Do we actually need this section ?

Reply:
I think many developers don't know about the ST BLE demo app.I removed the references to stm32wb55